code/game/jobs/job/engineering.dm

```diff
@@ -94,7 +94,7 @@
 	department_head = list("Chief Engineer")
 	selection_color = "#fff5cc"
 	access = list(ACCESS_EVA, ACCESS_ENGINE, ACCESS_ENGINE_EQUIP, ACCESS_TECH_STORAGE, ACCESS_MAINT_TUNNELS, ACCESS_EXTERNAL_AIRLOCKS, ACCESS_CONSTRUCTION, ACCESS_ATMOSPHERICS, ACCESS_MINERAL_STOREROOM)
-	minimal_access = list(ACCESS_EVA, ACCESS_ATMOSPHERICS, ACCESS_MAINT_TUNNELS, ACCESS_EXTERNAL_AIRLOCKS, ACCESS_EMERGENCY_STORAGE, ACCESS_CONSTRUCTION, ACCESS_MINERAL_STOREROOM, ACCESS_TECH_STORAGE)
+	minimal_access = list(ACCESS_EVA, ACCESS_ENGINE, ACCESS_ENGINE_EQUIP, ACCESS_ATMOSPHERICS, ACCESS_MAINT_TUNNELS, ACCESS_EXTERNAL_AIRLOCKS, ACCESS_EMERGENCY_STORAGE, ACCESS_CONSTRUCTION, ACCESS_MINERAL_STOREROOM, ACCESS_TECH_STORAGE)
```

Comment:
These additions give atmos techs access to Solars, Electrical Maintenance, Gravity Generator(!) and... the room with the Maintenance Drone machine.

Reply:
> These additions also give atmos techs access to Solars, Electrical Maintenance, Gravity Generator(!) and... the room with the Maintenance Drone machine, in addition to all of Engineering.

Hmm, gonna try a map change instead, as was proposed, or do a map change and something with this code, solars and elec maints with the grav gen should not all be on the same access